src/libsync/httplogger.cpp

```diff
@@ -121,14 +120,12 @@ void HttpLogger::logRequest(QNetworkReply *reply, QNetworkAccessManager::Operati
         timer->reset();
 
         const auto request = reply->request();
-        const auto keys = request.rawHeaderList();
-        QList<QNetworkReply::RawHeaderPair> header;
-        header.reserve(keys.size());
-        for (const auto &key : keys) {
-            header << qMakePair(key, request.rawHeader(key));
+        QJsonObject header;
+        for (const auto &key : request.rawHeaderList()) {
+            header[QString::fromUtf8(key)] = QString::fromUtf8(request.rawHeader(key));
```

Comment:
Doesn't this overwrite the authorize header you set above?

Reply:
This is the header object before its passed to logHttp